README.md

```diff
@@ -39,656 +39,17 @@ PLEASE NOTE that `Azuererm` subcommands are experimental and of alpha quality. N
 
 ## Configuration
 
-### ASM mode
-For this plugin to interact with Azure's REST API, you will need to give Knife
-information about your Azure account and credentials. The easiest way to do
-this is to sign in to the Azure portal and download a publishsettings file
-from https://manage.windowsazure.com/publishsettings/index?client=xplat to a
-local file system location, and
-then refer to the local file via an entry in your knife.rb:
+1. [ASM Mode] (docs/configuration.md#asm-mode)
+2. [ARM Mode] (docs/configuration.md#arm-mode)
 
-    knife[:azure_publish_settings_file] = "~/myazure.publishsettings"
+## Detailed Usage for ASM and ARM mode
 
-Alternatively, all subcommands for this plugin will accept an
---azure-publish-settings-file option to allow you to specify the path to that
-file with each command invocation.
+1. [ASM Mode] (docs/ASM.md)
+2. [ARM Mode] (docs/ARM.md)
 
-The plug-in also accepts authentication information specified using an
-alternative set of options -- see the section on "Alternative Management
-Certificate Specification" for details.
+## Bootstrap existing VM to install the chef-client using chef-extension :
 
-The plug-in can also read Azure account and credentials from the `Azure Profile` if Knife does not have the entry for `publish_settings_file`.
-An `Azure Profile` is a `JSON` file with subscription and environment information in it. Its default location is `~/.azure/azureProfile.json`.
+  We have added a utility in ASM and ARM to bootstrap existing VM. This will install the chef-client using chef extension on your VM.
```

Comment:
For ASM, we support `ssh` and `winrm` protocols too apart from `chef extension` for bootstrapping. Hence it's better to use just  `This will install the chef-client on your VM.` instead of `This will install the chef-client using chef extension on your VM.`